web/src/params/user.rs

```diff
@@ -47,3 +46,29 @@ impl IntoUpdateMap for UpdateParams {
         update_map
     }
 }
+
+#[derive(Debug, Deserialize, IntoParams, ToSchema)]
+pub struct UpdatePasswordParams {
```

Comment:
I love how closely the frontend and backend code follow each other here on types. This makes code comprehension so much easier.